sys-kernel/pf-sources/pf-sources-5.2_p1.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+ETYPE="sources"
+KEYWORDS="-* ~amd64 ~ppc ~ppc64 ~x86"
+
+HOMEPAGE="
+	https://gitlab.com/post-factum/pf-kernel/wikis/README
+	https://dev.gentoo.org/~mpagano/genpatches/
+"
+
+IUSE=""
+
+# No 'experimental' USE flag provided, but we still want to use genpatches
+K_EXP_GENPATCHES_NOUSE="1"
+
+# No reason to bump this number unless something new gets included in genpatches,
+# in that case we can manually remove the linux kernel patches from genpatches.
+K_GENPATCHES_VER="1"
+
+K_NOSETEXTRAVERSION="don't_set_it"
```

Comment:
Wouldn't simple `1` work?

Reply:
It should. Remnants of the old ebuild...